src/consensus/encode.rs

```diff
@@ -605,6 +605,7 @@ impl_vec!(Inventory);
 impl_vec!(Vec<u8>);
 impl_vec!((u32, Address));
 impl_vec!(u64);
+impl_vec!(FilterHeader);
```

Comment:
nit: may be it will be nice to have it right underneath `FilterHash` in line 600